features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/modules/analytics/list.jag

```diff
@@ -25,7 +25,8 @@ var getAnalyticsConfig = function (analyticsData) {
         client.setOptions(client_options);
 
         var eventingConfig = stub.getEventingConfigData();
-        //var analyzingConfigData = stub.getAnalyzingConfigData();
+        var analyzingConfigData = stub.getAnalyzingConfigData();         //Commented due to Current component not support this
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea